amgut/lib/data_access/ag_data_access.py

```diff
@@ -323,7 +340,10 @@ def deleteAGParticipantSurvey(self, ag_login_id, participant_name):
                 self.deleteSample(info['barcode'], ag_login_id)
 
             # Delete last due to foreign keys
-            sql = "DELETE FROM ag_kit_barcodes WHERE survey_id IN %s"
+            sql = "DELETE FROM ag.ag_kit_barcodes WHERE barcode IN %s"
```

Comment:
Actually it is bad. The call should've failed because there should've been a Foreign Key on the barcode. If it didn't fail before means that either the test is wrong or the FK is missing, both cases are bad. Please double-check.